src/ng/controller.js

```diff
@@ -81,16 +81,12 @@ function $ControllerProvider() {
      * It's just a simple call to {@link auto.$injector $injector}, but extracted into
      * a service, so that one can override this service with [BC version](https://gist.github.com/1649788).
      */
-    return function $controller(expression, locals, later, ident) {
+    return function $controller(expression, locals, _ignored, ident) {
```

Comment:
Theoretically, we can do whatever we want with "private", undocumented params :grin: